apps/mark-scan/integration-testing/cypress/support/e2e.js

```diff
@@ -89,7 +89,18 @@ function configureWithSampleDefinitionAndSystemSettings() {
   );
 }
 
+function fakeLoadPaper() {
+  cy.intercept('POST', 'http://localhost:3000/api/getPaperHandlerState', {
```

Comment:
Don't need the full URL.

```suggestion
  cy.intercept('POST', '/api/getPaperHandlerState', {
```